src/reducerFunction.js

```diff
@@ -0,0 +1,12 @@
+const reducerFunction = (state, action) => {
+    switch (action.type) {
+        case 'SHOW_COLOR_PALETTE': return { ...state, showColorPalette: action.payload }
```

Comment:
Keep return on a separate line. 

if you aren't using any code formatter, I would recommend to use one.